### PR TITLE
fixes/auth and scanning fixes

### PR DESCRIPTION
**Pull request by mhlangagc**
### Description:
- Fixes scanning issues picked up. 

### Pods Installed
 * Run `pod install` to remove the `SwipeKit` Library

### CODE REVIEW

File: Big Neon/Config/AnalyticsService.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import Foundation
+import Fabric
+import Crashlytics
+import Answers
+
+public enum ErrorType: String {
+    case authentication = "Authentication"
+    case eventFetching  = "Event Fetching"
+    case scanning       = "Scanning"
+    case guestlistFetch = "Fetching Guestlist"
```
Comment (Krakaw) [inline Big Neon/Config/AnalyticsService.swift]: Let's keep the nomenclature the same, so "Guest List" for all

File: Big Neon/Development.plist [hunk 2]
```diff
@@ -17,7 +19,21 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0.0</string>
 	<key>CFBundleVersion</key>
-	<string>150</string>
+	<string>180</string>
+	<key>Fabric</key>
+	<dict>
+		<key>APIKey</key>
+		<string>aa0b5f33b3ba00e613962388fc955505c3fe52d3</string>
```
Comment (Krakaw) [inline Big Neon/Development.plist]: Should this ever be public?

File: Big Neon/Production.plist [hunk 3]
```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>Fabric</key>
+    <dict>
+        <key>APIKey</key>
+        <string>aa0b5f33b3ba00e613962388fc955505c3fe52d3</string>
```
Comment (Krakaw) [inline Big Neon/Production.plist]: Should this be in a repository?

File: Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift [hunk 4]
```diff
@@ -143,9 +146,9 @@ extension GuestListViewController: SwipeActionTransitioning {
         if guestValues![indexPath.row].status == TicketStatus.purchased.rawValue {
             var options = SwipeOptions()
             options.backgroundColor = UIColor.brandPrimary
-            options.transitionStyle = .border
-            options.minimumButtonWidth = UIScreen.main.bounds.width * 0.3
-            options.expansionStyle = .selection
+            options.transitionStyle = .reveal
+//            options.minimumButtonWidth = UIScreen.main.bounds.width //* 0.4
+            options.expansionStyle = .fillReset(timing: .with) //.selection
```
Comment (Krakaw) [inline Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: Crashes compilation

File: Big Neon/Config/AnalyticsService.swift (hunk 1 above)
Comment (mhlangagc) [reply Big Neon/Config/AnalyticsService.swift]: Changed to Guest List & Checkin. Better to keep them as separate names for differentiated reporting in Fabric.

File: Big Neon/Development.plist (hunk 2 above)
Comment (mhlangagc) [reply Big Neon/Development.plist]: Removed from Repo

File: Big Neon/Production.plist (hunk 3 above)
Comment (mhlangagc) [reply Big Neon/Production.plist]: Removed from Repo

File: Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift (hunk 4 above)
Comment (mhlangagc) [reply Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: Was this running in XCode and building on a device? Works on my side across different devices!

Comment (Krakaw) [reply Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: image

Comment (Krakaw) [reply Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: Has this been fixed?

Comment (mhlangagc) [reply Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: Yes. It's now sorted.

**Comment by Krakaw** [comment]
image

File: Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift (hunk 4 above)
Comment (mhlangagc) [reply Big Neon/Screens/Explore/GuestListViewController/GuestListViewController.TableView.swift]: Please review this when you have time today.

**Pull request merged**